features/ajna/borrow/sidebars/AjnaBorrowFormContent.tsx

```diff
@@ -65,8 +67,8 @@ export function AjnaBorrowFormContent({
     ),
     primaryButton: {
       label: t(getPrimaryButtonLabelKey({ currentStep, product, dpmAddress, walletAddress })),
-      disabled: !!walletAddress && (!isStepValid || isAllowanceLoading),
-      isLoading: !!walletAddress && isAllowanceLoading,
+      disabled: !!walletAddress && (!isStepValid || isAllowanceLoading || isLoadingSimulation),
+      isLoading: (!!walletAddress && isAllowanceLoading) || isLoadingSimulation,
```

Comment:
Correct me if I'm wrong, but I think it should be
```suggestion
      isLoading: !!walletAddress && (isAllowanceLoading || isLoadingSimulation),
```
Unless wallet is connected we don't need to display loading, because that button leads only to connection page, not to next step.